yarn.lock

```diff
@@ -0,0 +1,19 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+nan@^2.12.1, nan@^2.14.0:
+  version "2.15.0"
+  resolved "https://registry.yarnpkg.com/nan/-/nan-2.15.0.tgz#3f34a473ff18e15c1b5626b62903b5ad6e665fee"
+  integrity sha512-8ZtvEnA2c5aYCZYd1cvgdnU6cqwixRoYg70xPLWUws5ORTa/lnw+u4amixRS/Ac5U5mQVgp9pnlSUnbNWFaWZQ==
+
+tree-sitter-cli@^0.19.1:
+  version "0.19.5"
+  resolved "https://registry.yarnpkg.com/tree-sitter-cli/-/tree-sitter-cli-0.19.5.tgz#8dbf73480223fff44559cec881c0cb1089a72faa"
+  integrity sha512-kRzKrUAwpDN9AjA3b0tPBwT1hd8N2oQvvvHup2OEsX6mdsSMLmAvR+NSqK9fe05JrRbVvG8mbteNUQsxlMQohQ==
+
+"tree-sitter-javascript@github:tree-sitter/tree-sitter-javascript#2c5b138":
+  version "0.19.0"
+  resolved "https://codeload.github.com/tree-sitter/tree-sitter-javascript/tar.gz/2c5b138ea488259dbf11a34595042eb261965259"
+  dependencies:
+    nan "^2.12.1"
```

Comment:
Do we need this `yarn.lock`?

Reply:
This is usually a committed file to keep actual package versions consistent. I can remove it , but I'd recommend having it in the repo.